tests/sqllogictests/src/main.rs

```diff
@@ -185,17 +187,23 @@ async fn run_suits(suits: ReadDir, client_type: ClientType) -> Result<()> {
                     .await
                     .unwrap();
             } else {
-                tasks.push(async move { run_file_async(&mut runner, file.unwrap().path()).await })
+                tasks.push(async move { run_file_async(&mut runner, file.unwrap().path()).await });
+                idx += 1;
+                if idx == parallel {
+                    // Run all tasks parallel
+                    run_parallel_async(tasks).await?;
```

Comment:
Better add a log here, reminding that a batch is complete.
```suggestion
                    run_parallel_async(tasks).await?;
                    println!("batch complete, progress: {done}/{all}");
```

Reply:
It does fix the problem and after running I think the current logging message is pretty reach for now. This can leave to anyone who is interested to add a progress bar or something.

Anyway, this PR does its job and should be merged ASAP. I'll approve it.